rootfs/api/models/release.py

```diff
@@ -218,6 +219,9 @@ def rollback(self, user, version=None):
                 raise DeisException('Cannot roll back to initial release.')
 
             prev = self.app.release_set.get(version=version)
+            if prev.failed:
+                raise DeisException('Cannot roll back to failed release.')
```

Comment:
The user always has a better idea whether or not he or she should roll back to a failed release. The fact that we are preventing them from re-deploying a "failed" release is frustrating to the user.